### PR TITLE
Use StringName consistently to refer to the Master audio bus name

### PR DESCRIPTION
**Pull request by Calinou**
These methods internally use StringName, so this should provide a minimal performance boost.

### CODE REVIEW

File: modules/minimp3/resource_importer_mp3.cpp [hunk 1]
```diff
@@ -33,6 +33,7 @@
 #include "core/io/file_access.h"
 #include "core/io/resource_saver.h"
 #include "scene/resources/texture.h"
+#include "scene/scene_string_names.h"
```
[the remark this answers is not in the document]
Comment (KoBeWi) [reply modules/minimp3/resource_importer_mp3.cpp:36]: If indirect includes are fine then likely not.

File: servers/audio_server.h [hunk 2]
```diff
@@ -36,6 +36,7 @@
 #include "core/os/os.h"
 #include "core/templates/safe_list.h"
 #include "core/variant/variant.h"
+#include "scene/scene_string_names.h"
```
Comment (YuriSizov) [inline servers/audio_server.h:39]: This should probably be in the CPP file.

**Review by YuriSizov — APPROVED** [review]
Aside from the notes that I've left, this seems like a good idea to me.

**Review by akien-mga — APPROVED** [review]
Seems fine.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**